functions/slack/SlackKnowledgeGraphSearch/Function.cs

```diff
@@ -0,0 +1,175 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Kgsearch.v1;
```

Comment:
It would be good if we could include the using somewhere. We do get issues around this often. Maybe a partial class with a file per method? Or similar to the verifier class, a searcher class and a formatter one, and inject those? I know it's ugly, but I can't think of anything else, and I really think it's better if everything gets included on the sample.

Reply:
I'd normally do that - but I was following the examples of the [existing tutorial](https://cloud.google.com/functions/docs/tutorials/slack) - the Java example doesn't have any imports.

If I use the same start/end multiple times, does that concatenate things together?

Reply:
Yes, that's something that I usually like to avoid because it leads to spaghetti code, but in this case it is justified and might be easiest I think.